addons/rookupgrade/10to14/install.sh

```diff
@@ -118,6 +129,12 @@ function rookupgrade_10to14_upgrade() {
 
             kubectl -n rook-ceph patch CephCluster rook-ceph --type=merge -p '{"spec": {"cephVersion": {"image": "ceph/ceph:v14.2.5-20201116"}}}'
             kubectl patch deployment -n rook-ceph csi-rbdplugin-provisioner -p '{"spec": {"template": {"spec":{"containers":[{"name":"csi-snapshotter","imagePullPolicy":"IfNotPresent"}]}}}}'
+
+            log "Awaiting up to 5 minutes to check Rook Ceph Pod(s) are Running"
+            if ! spinner_until 300 check_for_running_pods "rook-ceph"; then
+                logWarn "Rook Ceph has unhealthy Pod(s)"
+            fi
+
```

Comment:
Based on your PR description the problem seem to be coming from a timeout with: https://github.com/replicatedhq/kURL/blob/fa7ca211833e252769b684b4e761136bb9f49105/addons/rookupgrade/10to14/install.sh#L138

What makes you think this change will help with the `kurl rook wait-for-ceph-version` 20 min timeout?

Reply:
Know if the pods are running or not prior that mainly.

Reply:
@rrpolanco because we checking if all pods are running so that we will have further info when it is not.
Besides the timeout is big it is required for old versions.

Reply:
The WaitForRookOrCephVersion just look for  Rook or Ceph version.
We added the shell func check_for_running_pods to check if all pods in the namespace are running and OK 

We could convert the check_for_running_pods for a new func in go but I am not sure how much value it would bring at this point.

Reply:
Just to clarify:  the intention of this PR is we use the func that already exist and just warning if we find something that is not OK in the namespace and then output what is not OK.
So that if that does not fix the issue then, we can at least have further info in the logs. The idea is to be very low risk since those are old versions.